amazon-cloudwatch-container-insights/k8s-deployment-manifest-templates/deployment-mode/service/cwagent-prometheus/prometheus-k8s.yaml

```diff
@@ -45,26 +45,27 @@ data:
                 },
                 {
                   "source_labels": ["Service", "frontend"],
-                  "label_matcher": ".*haproxy-ingress-controller.*;(httpfront-shared-frontend|httpfront-default-backend|httpsfront)",
+                  "label_matcher": ".*haproxy-ingress-.*metrics;(httpfront-shared-frontend|httpfront-default-backend|httpsfront|_front_http)",
```

Comment:
Why do we need to change the matcher?

Reply:
The old helm chart repo is deprecated. After we migrate to the new repo, the service name changes from `haproxy-ingress-controller-metrics` to `haproxy-ingress-metrics`. This regex will match both of them.